mcs/class/System/common.sources

```diff
@@ -279,31 +279,35 @@ ReferenceSources/SR2.cs
 ReferenceSources/SRCategoryAttribute.cs
 ReferenceSources/Win32Exception.cs
 
-../referencesource/System/regex/system/text/regularexpressions/Regex.cs
```

Comment:
We had these modifications in Regex.cs:

https://github.com/mono/mono/blob/8ddcea295647203cf60f1e5f82a30297618ccd11/mcs/class/referencesource/System/regex/system/text/regularexpressions/Regex.cs#L1408-L1415

We should make sure to understand if the bug is still there in corefx.

Reply:
@akoeplinger there was no repro for that bug 🙁 And we already have corefx's tests for regular expressions and some of them test "Compile" logic - no failing tests so far